python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -218,10 +218,22 @@ class GbdesAstrometricFitConnections(
         storageClass="ArrowNumpyDict",
         dimensions=("instrument", "skymap", "tract", "physical_filter"),
     )
+    modelParams = pipeBase.connectionTypes.Output(
+        doc="WCS parameter covariance.",
+        name="gbdesAstrometricFit_modelParams",
+        storageClass="ArrowNumpyDict",
```

Comment:
TIL...

Reply:
About ArrowNumpyDicts?

Reply:
Yep, I don't see any other uses in the stack yet (probably because most devs also don't know it exists).

Reply:
Yeah, now that I'm thinking about it, this task's outputs are what triggered Eli adding this storage class.